regexp.go

```diff
@@ -19,7 +19,10 @@ import (
 )
 
 // Default timeout used when running regexp matches -- "forever"
-var DefaultMatchTimeout = time.Duration(math.MaxInt64)
+var (
+	DefaultMatchTimeout     = time.Duration(math.MaxInt64)
+	DefaultUnmarshalOptions = None
```

Comment:
nit: please add a usage comment on new package var and move DefaultMatchTimeout comment

Reply:
comment has been added.